src/AccessibilityInsights.SharedUx/Properties/Resources.Designer.cs

```diff
@@ -60,6 +60,15 @@ public class Resources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Built with Axe-Windows {0}.
```

Comment:
Product name in NuGet is "Axe.Windows", not "Axe-Windows"
```suggestion
        ///   Looks up a localized string similar to Built with Axe.Windows {0}.
```